views/legislation-page.js

```diff
@@ -192,10 +192,10 @@ const measureListRow = (s) => {
             <h3><a href="${measureUrl}">${s.title}</a></h3>
             ${s.introduced_at ? html`
             <div class="is-size-7 has-text-grey">
-              <span class="has-text-weight-bold">${s.short_id.replace(/^[^-]+-(\D+)(\d+)/, '$1 $2').toUpperCase()}</span> &mdash;
-              ${s.sponsor_first_name
-                ? html`Introduced by&nbsp;<a href=${`/${s.sponsor_username}`}>${s.sponsor_first_name} ${s.sponsor_last_name}</a>&nbsp;on ${(new Date(s.introduced_at)).toLocaleDateString()}`
-                : html`Introduced on ${(new Date(s.introduced_at)).toLocaleDateString()}`
+              <span class="has-text-weight-bold">${s.short_id.replace(/^[^-]+-(\D+)(\d+)/, '$1 $2').toUpperCase()}</span>
+                ${s.sponsor_first_name
+                ? html`&mdash; <a href=${`/${s.sponsor_username}`}>${s.sponsor_first_name} ${s.sponsor_last_name}</a> &bullet; <a href="${subjectUrl(url, s.policy_area)}">${s.policy_area}</a>${s.policy_area ? ' • ' : ''}${(new Date(s.introduced_at)).toLocaleDateString()}`
```

Comment:
Shouldn't the `<a href="${subjectUrl(url, s.policy_area)}">${s.policy_area}</a>` code be within the `${s.policy_area ? ' • ' : ''}` ternary? This looks like it will still show a policy area link even if null.

Reply:
If it's null, it shows nothing. Here's a congressional bill with no policy_area

I tried to put it inside the ternary, but the link stopped working. 
![image](https://user-images.githubusercontent.com/39286778/56965437-ee256900-6b22-11e9-95e0-b124e1b6da71.png)